string name
float64[] goal_pose
float64[] belief_updated
